roles/common/defaults/main.yml

```diff
@@ -35,6 +35,10 @@ common:
     tar_version: master
   system_tools:
     mcelog: True
+  service:
+    name: mcelog
+    state: restarted
+    enabled: True
```

Comment:
Can you please let me know the correct place to put it?  mcelog is not currently being started on boot and needs to be to catch memory errors in /var/log/mcelog